cockatrice/src/game/zones/view_zone_widget.cpp

```diff
@@ -33,6 +40,7 @@ ZoneViewWidget::ZoneViewWidget(Player *_player,
     QGraphicsLinearLayout *vbox = new QGraphicsLinearLayout(Qt::Vertical);
     QGraphicsLinearLayout *hPilebox = 0;
 
+    // If the number is < 0 then it means that we can make the area sorted and we dont care about the order.
```

Comment:
when the entire deck is shown we still have the option to show it unsorted, it's sorted by default yeah